config/generate/kinto.tpl

```diff
@@ -11,20 +11,20 @@ use = egg:kinto
 # Backends.
```

Comment:
Rather than moving the existing kinto.ini file somewhere else, let it where it is currently and create a new kinto.tpl file, like you did.